python/cudf/cudf/core/multiindex.py

```diff
@@ -1302,6 +1302,54 @@ def _poplevels(self, level):
 
         return popped
 
+    @_cudf_nvtx_annotate
+    def swapij(self, i=0, j=1):
```

Comment:
[pandas has a `MultiIndex.swaplevel` function](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.MultiIndex.swaplevel.html), is there a reason that you chose a different name here? Also, all the comments from above about the default values and the docstring still apply here.

I would perhaps recommend defining this docstring as a module-level variable and then importing it into `dataframe.py` and using our `docutils.doc_apply` decorator. That way you can share the description between the MultiIndex and DataFrame methods and only add in the specialized examples/return value. [Here is an example of how we do that in the code](https://github.com/rapidsai/cudf/blob/branch-22.08/python/cudf/cudf/core/dataframe.py#L2598).